Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3169,6 +3169,53 @@ public void onActivityDestroyed(Activity activity) {
 
     /**
      * <p>An Interface class that is implemented by all classes that make use of
+<<<<<<< Updated upstream
+=======
+     * {@link BranchLinkShareListener}, defining methods to listen for link sharing status.</p>
+     */
+    public interface BranchLinkShareListener2 {
```

Comment:
Name `BranchLinkShareListener2` seems like this is a dead code added from merge with local copy